<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OxygenExceptionWidget</class>
 <widget class="QWidget" name="OxygenExceptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>352</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window Identification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Matching window property: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>exceptionType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Regular expression to match: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>exceptionEditor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="detectDialogButton">
        <property name="text">
         <string>Detect Window Properties</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="KLineEdit" name="exceptionEditor">
        <property name="showClearButton" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="KComboBox" name="exceptionType">
        <item>
         <property name="text">
          <string>Window Class Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Window Title</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Decoration Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="frameBorderCheckBox">
        <property name="text">
         <string>Border size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="sizeGripCheckBox">
        <property name="text">
         <string>Add handle to resize windows with no border</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="titleOutlineCheckBox">
        <property name="text">
         <string>Outline active window title:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="separatorCheckBox">
        <property name="text">
         <string>Separator display:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="blendColorCheckBox">
        <property name="text">
         <string>Background style:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QCheckBox" name="hideTitleBar">
        <property name="text">
         <string>Hide window title bar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="KComboBox" name="frameBorderComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">No Border</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">No Side Borders</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Tiny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Very Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Huge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Very Huge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Oversized</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="KComboBox" name="blendColorComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Solid Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Radial Gradient</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Follow Style Hint</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="KComboBox" name="sizeGripComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string comment="Add handle to resize windows with no border">Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="Add handle to resize windows with no border">Enabled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="KComboBox" name="titleOutlineComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string comment="outline window title">Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="outline window title">Enabled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="KComboBox" name="separatorComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Never Draw Separator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Draw Separator When Window is Active</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always Draw Separator</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>exceptionType</tabstop>
  <tabstop>exceptionEditor</tabstop>
  <tabstop>detectDialogButton</tabstop>
  <tabstop>frameBorderCheckBox</tabstop>
  <tabstop>frameBorderComboBox</tabstop>
  <tabstop>blendColorCheckBox</tabstop>
  <tabstop>blendColorComboBox</tabstop>
  <tabstop>sizeGripCheckBox</tabstop>
  <tabstop>sizeGripComboBox</tabstop>
  <tabstop>titleOutlineCheckBox</tabstop>
  <tabstop>titleOutlineComboBox</tabstop>
  <tabstop>separatorCheckBox</tabstop>
  <tabstop>separatorComboBox</tabstop>
  <tabstop>hideTitleBar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>frameBorderCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameBorderComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blendColorCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>blendColorComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeGripCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeGripComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleOutlineCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>titleOutlineComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>separatorCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>separatorComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
